fhirResourcesToLoad/o2teddy_observation_ahi.json

```diff
@@ -2,6 +2,18 @@
   "resourceType": "Observation",
   "id": "obs014-ahi",
   "status": "final",
+  "category": [
+    {
+      "coding": [
+        {
+          "system": "http://terminology.hl7.org/CodeSystem/observation-category",
+          "code": "laboratory",
+          "display": "Laboratory"
+        }
+      ],
+      "text": "Laboratory"
+    }
+  ],
```

Comment:
code system for line 39 is 'http://terminology.hl7.org/CodeSystem/v2-0078'

Reply:
How did you figure out?  👍